dataforest/config/MetaPlotMethods.py

```diff
@@ -9,4 +10,19 @@ def PLOT_METHOD_LOOKUP(cls):
 
     @property
     def PLOT_METHODS(cls):
-        return cls.CONFIG["plot_methods"]
+        try:
+            plot_methods = cls.CONFIG["plot_methods"]
```

Comment:
Another way to write this would be `plot_methods = cls.CONFIG.get("plot_methods", parse_plot_methods(config=cls.CONFIG))`

Reply:
Couldn't find `parse_plot_methods`, is this outdated?